pkg/security/tests/filters_test.go

```diff
@@ -222,7 +222,7 @@ func TestOpenParentDiscarderFilter(t *testing.T) {
 	// a discarder is created).
 	rule := &rules.RuleDefinition{
 		ID:         "test_rule",
-		Expression: `open.file.path =~ "/usr/local/no-approver-*" && open.flags & (O_CREAT | O_SYNC) > 0`,
+		Expression: `open.file.path =~ "{{.Root}}/no-approver-*" && open.flags & (O_CREAT | O_SYNC) > 0`,
```

Comment:
I think that for this test we should use a sub folder of `{{.Root}}/test/test-obd-2*` to have the file matching but not the parent, because the purpose was to test that event with a path with the name matching we can have a discarder thanks to the parent.